content/docs/pipelines/upgrade.md

```diff
@@ -0,0 +1,218 @@
++++
+title = "Upgrading and Reinstalling"
+description = "How to upgrade or reinstall your Kubeflow Pipelines deployment"
+weight = 20
++++
+
+Starting from version TODO:VERSION-NUMBER, Kubeflow Pipelines persists the
```

Comment:
This will land in Kubeflow v0.5. I would recommend call out it's kubeflow version
"Starting from Kubeflow version v0.5..."

Reply:
Done.